components/eam/cime_config/config_compsets.xml

```diff
@@ -227,48 +178,27 @@
   <!-- MMF compsets (i.e. super-parameterization) -->
   <!-- ****************************************** -->
 
-  <!-- ********** F compsets ********** -->
+  <!-- F compsets with climatological SST -->
   <compset>
-    <alias>F-MMF1</alias>
+    <alias>F-2010-MMF1</alias>
```

Comment:
For EAM compsets, we combine this into `F2010`; I think it would be better to follow this with MMF, so this would be `F2010-MMF`.